desc/objectives/objective_funs.py

```diff
@@ -310,6 +311,35 @@ def compile(self, mode="auto", verbose=1):
             timer.disp("Total compilation time")
         self._compiled = True
 
+    def is_unnested(
```

Comment:
does this need to be part of the objective class? I think it could just be a callback thats created in the optimizer